dockerfiles/examples/local/hpcc-localfile/templates/NOTES.txt

```diff
@@ -2,10 +2,10 @@
 {{ $daliPvcName := printf "dali-%s-pvc" (include "hpcc-localfile.fullname" $) }}
 {{ $dataPvcName := printf "data-%s-pvc" (include "hpcc-localfile.fullname" $) }}
 
-Persistent Volume Claims created: {{ $dllPvcName }}, {{ $daliPvcName }}, {{ $dataPvcName }}
-
-Pass a values yaml to hpcc helm charts to use these existing claims.
-e.g.:
+# Persistent Volume Claims created: {{ $dllPvcName }}, {{ $daliPvcName }}, {{ $dataPvcName }}
+#
+# Pass a values yaml to hpcc helm charts to use these existing claims.
+# e.g.:
```

Comment:
agree good idea to comment these, is it worth amending the other examples to same in this PR?

Reply:
I think use a separate PR.  We should review the others in the light of this PR.